app/assets/stylesheets/messages/base.css.scss

```diff
@@ -0,0 +1,17 @@
+.message {
+  text-align: left;
+  margin-top: 1em;
+  margin-left: 3em;
+
+  .message-header {
+
+    .button{
+      display: inline;
+      float: center;
+    }
+
+    .message-content {
+      padding: 10px 10px 10px 10px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `10px` instead of `10px 10px 10px 10px`